pkg/kclient/deployments.go

```diff
@@ -0,0 +1,37 @@
+package kclient
+
+import (
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateDeployment creates a deployment based on the given pod
+func (c *Client) CreateDeployment(pod *corev1.Pod) (*appsv1.Deployment, error) {
+
+	replicas := int32(1)
+	deployment := appsv1.Deployment{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Deployment",
+			APIVersion: "apps/v1",
```

Comment:
should we make this a const?

Reply:
Added constants for the Kind and APIVersion